shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/context/PipelineJobContext.java

```diff
@@ -66,4 +69,25 @@ public interface PipelineJobContext {
      * @return job process context
      */
     PipelineProcessContext getJobProcessContext();
+    
+    /**
+     * Get task configuration.
+     *
+     * @return task configuration
+     */
+    TaskConfiguration getTaskConfig();
+    
+    /**
+     * Get init progress.
+     *
+     * @return init progress
+     */
+    JobProgress getInitProgress();
+    
+    /**
+     * Get source data source.
+     *
+     * @return source data source
+     */
+    PipelineDataSourceWrapper getSourceDataSource();
```

Comment:
It's better not put them in `PipelineJobContext`.
We could transfer these fields to InventoryTaskSplitter, or add them in sub-interface of PipelineJobContext.